src/pygambit/game.pxi

```diff
@@ -672,16 +672,20 @@ class Game:
                     mspr[s] = Rational(v)
             return mspr
 
-    def mixed_behavior_profile(self, rational=False) -> MixedBehaviorProfile:
-        """Create a behavior strategy profile over the game.
-
-        The profile is initialized to uniform randomization for each player
-        over their actions at each information set.
+    def mixed_behavior_profile(self, data=None, rational=False) -> MixedBehaviorProfile:
+        """Returns a behavior strategy profile `MixedBehaviorProfile` over the game,
+        initialized to uniform randomization for each player over his actions at each
```

Comment:
The docstring repeats itself; it should say what happens when `data` are specified.